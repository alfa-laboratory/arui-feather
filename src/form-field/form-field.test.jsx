/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */

import React from 'react';
import { mount } from 'enzyme';

import FormField from './form-field';

describe('form-field', () => {
    it('should render without problems', () => {
        let formField = mount(<FormField>FormField-test</FormField>);

        expect(formField).toMatchSnapshot();
    });
});
