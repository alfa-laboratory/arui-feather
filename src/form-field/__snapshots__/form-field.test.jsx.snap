// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`form-field should render without problems 1`] = `
<FormField
  size="m"
>
  <div
    className="form-field form-field_size_m form-field_theme_alfa-on-white"
  >
    FormField-test
  </div>
</FormField>
`;
